engines/librex/fallback.php

```diff
@@ -62,7 +62,7 @@ function get_librex_results($opts) {
 
             if (!empty($results)) {
                 $results["results_source"] = parse_url($instance)["host"];
-                error_log($results["results_source"]);
+                error_log("Empty Results from: " . $results["results_source"]);
```

Comment:
This error log statement was probably leftover from some debugging session, so unless there is another reason, I'd say its fine to just be removed entirely.

Reply:
I can't figure out how to replace that commit with one that removes it at the moment, so at the moment I've just pushed a new commit deleting that line entirely.